src/test/java/gov/healthit/chpl/aqa/asserts/ListingHistoryEyeAsserts.java

```diff
@@ -0,0 +1,64 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.junit.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.model.ListingHistoryData;
+import gov.healthit.chpl.aqa.pageObjects.ListingHistoryEyePage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+
+/**
+ * Class ListingHistoryEyeAsserts definition.
+ */
+public class ListingHistoryEyeAsserts extends Base {
+
+    /**
+     * Assert that expected product edit details is displayed on product history eye.
+     * @param expectedEditDetails is "Developer/Product/Version change"
+     * @throws Exception of type ParseException or InterruptedException
+     */
+    @Then("^I see the listing history eye shows a \"([^\"]*)\" on today's date$")
+    public void iSeeListingHistory(final String expectedEditDetails) throws Exception {
+        Thread.sleep(SLEEP_TIME);
+        String actualListingHistoryDetails = ListingHistoryEyePage.listingHistoryTable(getDriver()).getText();
+        String[] listingHistoryDetails = actualListingHistoryDetails.split("\n");
+        int activityCount = 0;
+        ListingHistoryData listingHistoryDataInst = null;
+        String date = null;
+        for (String activity : listingHistoryDetails) {
+            if ((activityCount % 2) == 0) {
+                date = activity;
+            } else {
+                if (activity.startsWith(expectedEditDetails)) {
+                    listingHistoryDataInst = new ListingHistoryData();
+                    listingHistoryDataInst.setDate(date);
+                    listingHistoryDataInst.setActivity(activity);
```

Comment:
You never use this Activity data later. I don't think adding the model to this helps in any real way